Palo Alto Networks/paloalto-cortex/_meta/smart-descriptions.json

```
@@ -0,0 +1,33 @@
+[
+  {
+    "value": "{cortexxdr.category}: An event of {cortexxdr.severity_human} severity was {cortexxdr.action} on endpoint {cortexxdr.shost}, initiated by user {cortexxdr.suser.0}.",
+    "conditions": [
+      {
+        "field": "cortexxdr.category"
+      },
+      {
+        "field": "cortexxdr.severity_human"
+      },
+      {
+        "field": "cortexxdr.action"
+      },
+      {
+        "field": "cortexxdr.shost"
+      },
+      {
+        "field": "cortexxdr.suser.0"
+      }
+    ]
+  },
+  {
+    "value": "{cortexxdr.category}: {cortexxdr.original_message}",
```

The original message already exists in the field `message`
```suggestion
    "value": "{cortexxdr.category}: {message}",
```